index.html

```diff
@@ -10,7 +10,54 @@
     <link rel="stylesheet" href="styles/index.css" />
 </head>
 <body>
-    <script src="./src/index.js"></script>
+    <header class="header">
+        <span>
+            <h1>Weather Report</h1>
+            <h2>For the city of
+                <h3 id="cityname">Seattle</h3>
+            </h2>
+        </span>
```

Comment:
```suggestion
            <h1>Weather Report</h1>
            <h2>For the city of
                <span id="cityname">Seattle</span>
            </h2>
```
Just some small changes to which tags you're using:
* The outermost `<span>` tag isn't necessary so we can get rid of it.
* An `<h#>` tag should denote the start of a new content section so I'd recommend only using the `<h1>` tag here. I can see an argument for having "For the city of Seattle" in an `<h2>` tag so I left it, but I would probably use a `<p>` tag instead.
* I wrapped "Seattle" in a `<span>` tag since this tag is just here to allow for additional styling and JS events, making `<span>` the most appropriate.